Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2585 @@
+        'md5': md5Regex
+    }
+    if indicator_type not in hash_to_regex:
+        raise ValueError(f'Indicator type {indicator_type} is not supported')
+
+    if not re.match(hash_to_regex[indicator_type], indicator_value):
+        raise ValueError(f'{indicator_value} is not a valid {indicator_type}')
+
+    return True
+
+
+def get_incident_uuid(client: Client, args: dict[str, Any]) -> str | None:
+    """
+      Get the incident UUID
+      Args:
+          client: client object to use.
+          args: all command arguments, usually passed from ``demisto.args()``.
+      Returns:
+        Return Incident UUID
+    """
+    payload = create_content_query(args)
+
+    # offset does not support by API therefore need to be removed
+    payload.pop('offset')
+
+    # search query as Lucene query string
+    if search_query := get_incident_filter_query(args):
+        payload['query'] = search_query
+
+    if not (
+        raw_data := client.get_incident(payload).get('result', [])
+    ):
+        raise DemistoException(f'Incident ID {args.get("incident_id")} was not found. '
+                               f'If it\'s older than 30 days, try increasing the time range arguments')
+    return raw_data[0].get('uuid')
+
+
+def create_payload_for_query(args: dict[str, Any], query_type: Optional[str] = None) -> dict:
+    """
+    Create payload for request the endpoints
+    Args:
+        args: all command arguments, usually passed from ``demisto.args()``.
+        query_type: query type : association, event, incident, allow_list, deny_list
+    Returns:
+        payload (dict): Return payload for request body
+    """
+    if query_type in ('allow_list', 'deny_list'):
+        limit = arg_to_number(args.get('limit'))
+        page_size = arg_to_number(args.get('page_size'))
+        if (limit and limit < 10) or (page_size and page_size < 10):
+            raise ValueError('Invalid input limit or page_size. '
+                             'For the Deny and Allow list specify the limit/page_size range '
+                             'The value must be >= 10 and <= 1000.')
+        payload = create_params_query(args)
+    else:
+        payload = create_content_query(args)
+
+    # search query as Lucene query string
+    if query_type == 'association':
+        search_query = get_association_filter_query(args)
+    elif query_type == 'event':
+        search_query = get_event_filter_query(args)
+    elif query_type == 'incident':
+        search_query = get_incident_filter_query(args)
+    else:
+        # default
+        search_query = args.get('query', '')
+
+    if search_query:
+        payload['query'] = search_query
+
+    return payload
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def common_wrapper_command(client_func: Callable, cmd_args: dict, readable_title: str,
+                           context_path: str, output_key_field: str, command_type: str = None,
+                           is_call_diff_readable_output: bool = False,
```

Comment:
no need for the `is_call_diff_readable_output` variable, pass `None` instead (and later, `if func_readable_output is not None ...` instead of `is_call_diff_readable_output`)